### PR TITLE
[-help] option in man pages

### PR DESCRIPTION
**Pull request by ajmohan**
added [-help] option in the documentation of apps commands

### CODE REVIEW

**Comment by ajmohan** [comment]
[-help] option in most of the commands documentation was missing and in ciphers and rehash commands it was wrongly specified as [-h], which is not considered as a valid option.

File: doc/apps/ciphers.pod [hunk 1]
```diff
@@ -18,6 +18,7 @@ B<openssl> B<ciphers>
 [B<-psk>]
 [B<-stdname>]
 [B<cipherlist>]
+[B<-help>]
```
Comment (mattcaswell) [inline doc/apps/ciphers.pod]: Probably this should be listed before the cipherlist

**Comment by mattcaswell** [comment]
One minor tweak required. Otherwise looks good.

**Comment by ajmohan** [comment]
Agreed, moved [-help] option before [cipherlist].

**Comment by mattcaswell** [comment]
+1

**Comment by ajmohan** [comment]
Thanks for the review.

File: doc/apps/ciphers.pod (hunk 1 above)
Comment (richsalz) [inline doc/apps/ciphers.pod]: I think we should consistently put -help as the first option listed.

**Comment by ajmohan** [comment]
Sure, consistently keeping [-help] as first option will look neat. Done, thanks.

**Comment by richsalz** [comment]
+1/  @mattcaswell you okay with this?  if so I'll pull it in.

**Comment by mattcaswell** [comment]
+1

**Comment by richsalz** [comment]
done in 0ae9e2926654657862e104a111a4e3028b0be8f6  thanks!

**Comment by ajmohan** [comment]
@richsalz:  0ae9e29 has only 3 files modified. I am not seeing all the changes on master. For example, ca.pod, crl.pod, dgst.pod, etc do not have -help option. If there are still some commits pending, never mind. Thanks.

**Comment by richsalz** [comment]
yeah still pending :)

**Pull request closed without merging**